tools/rails.md

```diff
@@ -0,0 +1,31 @@
+---
+permalink: /rails
+layout: post
+title: Rails
+link: https://rubyonrails.org/
```

Comment:
Can you change this to a more specific page that has the releases listed \ release policy please?

Reply:
Changed to a page that has the release policy. There are several other pages that might be useful: https://weblog.rubyonrails.org/releases/ for releases and https://rubygems.org/gems/rails/versions.

Reply:
You can link these in the page content.

Reply:
Done, please take another look.